test/config.yaml

```diff
@@ -0,0 +1,86 @@
+#### General Settings ####
+# location of system level settings
+settings:
+  title: My Very Cool Project
+  Author: Bob
+
+  # path like settings
+  data: /data/bob/original_data
+
+  # conda environment names
+  python2: py2.7
+
+  # names to access specific envs by
+  env: HOME
+
+#### Experiment Level Settings ####
+# experiment level settings, settings that apply to all samples
+exp.settings:
+
+  # Sample information relating sample specific settings to sample ids
+  sampleinfo: sample_metadata.csv
+
+  # it would be nice to be able define a setting here that applies to all
+  # samples, or define for each sample in the sampleinfo table case they are
+  # different.
+  fastq_suffix: '.fastq.gz'
+  # Need to some way to specify annotation to use, maybe here is not the best
+  # place.
+  annotation:
```

Comment:
How to handle cases where we want to try multiple annotations? For example, to compare a truncated gene model annotation with the full.  Maybe the answer there is just symlink over the upstream files to a new analysis dir rather than increase complexity here.